receiver/kafkareceiver/config.go

```diff
@@ -70,6 +73,11 @@ type Config struct {
 	MessageMarking MessageMarking `mapstructure:"message_marking"`
 }
 
+const (
+	OffsetLatest   string = "latest"
```

Comment:
are these needed outside of this package? If not do they need  to be public?

Reply:
@codeboten you are right. Its just needed within the package. I change the scope to private.